### ISSUE TITLE
Stylelint not reporting any errors on Preact project

### ISSUE THREAD

**Issue by lipe-dev** [stylelint/postcss-css-in-js#250]
### What steps are needed to reproduce the bug?

Running stylelint against any files doesn't generate any errors, nor does --fix do anything.

This is the same config I use in all my other projects, and they work fine there.

I can reproduce it without the order and prettier plugins as well, I can basically write any CSS and it will allow it (for either tsx or .css files)

### What Stylelint configuration is needed to reproduce the bug?

```js
const sortOrderSmacss = require("stylelint-config-property-sort-order-smacss/generate");

module.exports = {
  extends: [
    "stylelint-config-recommended",
    "stylelint-config-property-sort-order-smacss",
    "stylelint-config-prettier",
  ],
  plugins: ["stylelint-order"],
  customSyntax: "@stylelint/postcss-css-in-js",
  rules: {
    "order/properties-order": [
      sortOrderSmacss({ noEmptyLineBetween: true, emptyLineBefore: "always" }),
    ],
  },
  overrides: [
    {
      files: ["**/*.{css,js,jsx,ts,tsx}"],
    },
  ],
};
```

### How did you run Stylelint?

`yarn stylelint src/components/navbar/index.tsx` and also `yarn stylelint src/style/index.css`

### Which version of Stylelint are you using?

14.1.0

### What did you expect to happen?

Some errors should be reported (I basically broke the CSS code in every way I could imagine)

### What actually happened?

Nothing. It executes and finishes without any errors as if the files were OK.

### Does the bug relate to non-standard syntax?

I tried it both with (styled-components) and without (removing custom syntax and running against .css files)

### Proposal to fix the bug

No idea

**Comment by hudochenkov** [comment]
Should be fixed in just released `0.37.3`.
